tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt

```diff
@@ -2,388 +2,282 @@
-Oracle	;	ru	;	modern
-Oracle	;	sv	;	modern
-Oracle	;	zh	;	modern
-Oracle	;	zh_Hant	;	modern
-Oracle	;	af	;	basic
-Oracle	;	az	;	basic
-Oracle	;	be	;	basic
-Oracle	;	bg	;	basic
-Oracle	;	bs	;	basic
-Oracle	;	ca	;	basic
-Oracle	;	cy	;	basic
-Oracle	;	da	;	basic
-Oracle	;	el	;	basic
-Oracle	;	et	;	basic
-Oracle	;	fi	;	basic
-Oracle	;	ga	;	basic
-Oracle	;	he	;	basic
-Oracle	;	hi	;	basic
-Oracle	;	hr	;	basic
-Oracle	;	hu	;	basic
-Oracle	;	id	;	basic
-Oracle	;	is	;	basic
-Oracle	;	km	;	basic
-Oracle	;	lt	;	basic
-Oracle	;	lv	;	basic
-Oracle	;	mk	;	basic
-Oracle	;	ml	;	basic
-Oracle	;	mt	;	basic
-Oracle	;	nn	;	basic
-Oracle	;	no	;	basic
-Oracle	;	ro	;	basic
-Oracle	;	sk	;	basic
-Oracle	;	sl	;	basic
-Oracle	;	sq	;	basic
-Oracle	;	sr	;	basic
-Oracle	;	ta	;	basic
-Oracle	;	te	;	basic
-Oracle	;	th	;	basic
-Oracle	;	tr	;	basic
-Oracle	;	uk	;	basic
-Oracle	;	vi	;	basic
-
-openoffice_org	;	ar	;	basic
-openoffice_org	;	cs	;	basic
-openoffice_org	;	de	;	basic
-openoffice_org	;	en	;	basic
-openoffice_org	;	es	;	basic
-openoffice_org	;	fr	;	basic
-openoffice_org	;	it	;	basic
-openoffice_org	;	ja	;	basic
-openoffice_org	;	ko	;	basic
-openoffice_org	;	nl	;	basic
-openoffice_org	;	pl	;	basic
-openoffice_org	;	pt	;	basic
-openoffice_org	;	ru	;	basic
-openoffice_org	;	sv	;	basic
-openoffice_org	;	zh	;	basic
-openoffice_org	;	zh_Hant	;	basic
+
 openoffice_org	;	af	;	basic
-openoffice_org	;	az	;	basic
-openoffice_org	;	be	;	basic
-openoffice_org	;	bg	;	basic
-openoffice_org	;	bs	;	basic
-openoffice_org	;	ca	;	basic
-openoffice_org	;	cy	;	basic
-openoffice_org	;	da	;	basic
-openoffice_org	;	el	;	basic
-openoffice_org	;	et	;	basic
-openoffice_org	;	fi	;	basic
-openoffice_org	;	ga	;	basic
-openoffice_org	;	he	;	basic
-openoffice_org	;	hi	;	basic
-openoffice_org	;	hr	;	basic
-openoffice_org	;	hu	;	basic
-openoffice_org	;	id	;	basic
-openoffice_org	;	is	;	basic
-openoffice_org	;	km	;	basic
-openoffice_org	;	lt	;	basic
-openoffice_org	;	lv	;	basic
-openoffice_org	;	mk	;	basic
-openoffice_org	;	ml	;	basic
-openoffice_org	;	mt	;	basic
-openoffice_org	;	nn	;	basic
-openoffice_org	;	no	;	basic
-openoffice_org	;	ro	;	basic
-openoffice_org	;	sk	;	basic
-openoffice_org	;	sl	;	basic
-openoffice_org	;	sq	;	basic
-openoffice_org	;	sr	;	basic
-openoffice_org	;	ta	;	basic
-openoffice_org	;	te	;	basic
-openoffice_org	;	th	;	basic
-openoffice_org	;	tr	;	basic
-openoffice_org	;	uk	;	basic
-openoffice_org	;	vi	;	basic
-openoffice_org	;	nr	;	basic
-openoffice_org	;	nso	;	basic
-openoffice_org	;	ss	;	basic
-openoffice_org	;	st	;	basic
 openoffice_org	;	tn	;	basic
-openoffice_org	;	ts	;	basic
-openoffice_org	;	ve	;	basic
-openoffice_org	;	xh	;	basic
-openoffice_org	;	zu	;	basic
 
 #http://w3-03.ibm.com/globalization/page/4154
 
-IBM	;	en	;	comprehensive
-# IBM Group 1
-IBM	;	zh	;	modern
-IBM	;	zh_Hant	;	modern
-IBM	;	ja	;	modern
-IBM	;	ko	;	modern
-IBM	;	fr	;	modern
-IBM	;	de	;	modern
-IBM	;	it	;	modern
-IBM	;	es	;	modern
-IBM	;	pt	;	modern
-# IBM Group 2
+IBM	;	am	;	moderate
```

Comment:
are the IBM ones just rearranged (and the lowercase 'ibm' lines added)?

Reply:
yes